### PR TITLE
feat: get active and disabled objects from backend in Objects Setting…

### LINKED ISSUE #2005: Object Settings - Get object table data from backend
Use the new hooks to get the standard and custom objects from backend.

### PR DESCRIPTION
**Pull request by thaisguigon**
…s page

Closes #2005

### CODE REVIEW

File: front/src/pages/settings/data-model/SettingsObjects.tsx [hunk 1]
```diff
@@ -54,6 +37,19 @@ export const SettingsObjects = () => {
   const theme = useTheme();
   const navigate = useNavigate();
 
+  const activeMetadataObjects = useRecoilValue(activeMetadataObjectsSelector);
```
Comment (charlesBochet) [inline front/src/pages/settings/data-model/SettingsObjects.tsx:40]: Could we move this to a hook that would expose the metadata state without having the external users to know about the state names and internal implementation.

I would love to have something like: const { activeObjects, disableObjects ) = useObjectMetadata() that is internally doing these calls

**Review by charlesBochet — APPROVED** [review]
Approved with comments!

File: front/src/pages/settings/data-model/SettingsObjects.tsx (hunk 1 above)
Comment (thaisguigon) [reply front/src/pages/settings/data-model/SettingsObjects.tsx]: Done!

**Review by charlesBochet — APPROVED** [review]
LGTM!

**Pull request merged**